ui/src/components/InOutSwitch/InOutSwitch.jsx

```diff
@@ -1,110 +1,55 @@
-/* eslint-disable react/jsx-handler-names */
 import React from 'react';
 import { Badge, Button, OverlayTrigger, Popover } from 'react-bootstrap';
-import './style.css';
+import styles from './inOutStyle.module.css';
 
 export default class InOutSwitch extends React.Component {
```

Comment:
We should maybe make the effort to convert this component to a functional component